CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and all PyGraphistry-specific breaking changes are explictly noted here.
 
 ## [Development]
+### Added
```

Comment:
afaict the main change is allowing parameterization of the impute method (across all columns) + changing something about similarities